canonical-web-and-design/ubuntu.com
# Delete redundant global nav partial

## Done

Delete redundant global nav partial 

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Verify this partial is not used anywhere in the site
